docs/typescript/reference/transactapi/workflow-queues.md

```diff
@@ -16,20 +16,22 @@ interface QueueRateLimit {
     periodSec: number;
 }
 
+interface QueueParameters {
+    workerConcurrency?: number;
+    concurrency?: number;
+    rateLimit?: QueueRateLimit;
+}
+
 class WorkflowQueue {
-    constructor(
-        readonly name: string,
-        readonly concurrency?: number,
-        readonly rateLimit?: QueueRateLimit
-    );
+   constructor(name: string, queueParameters: QueueParameters);
```

Comment:
Is it a breaking change or do we still support the old way?

Reply:
Not breaking -- the change is backward compatible. See https://github.com/dbos-inc/dbos-transact-ts/blob/98676abe6c8c91084243c63279ca8f706b0ed108/src/wfqueue.ts